components/merkledb/CHANGELOG.md

```diff
@@ -0,0 +1,75 @@
+# Changelog
```

Comment:
@pavel-mukhanov I suppose this content should be moved to the main CHANGELOG file ?